packages/web/src/fixtures/withApollo/index.tsx

```diff
@@ -22,7 +22,7 @@ const isBrowser = typeof window !== 'undefined'
 export default withApollo<NormalizedCacheObject>(
   ({ ctx, headers }) => {
     const withHttp = createHttpLink({
-      uri: `${process.env.API_ORIGIN}/graphql`, // Server URL (must be absolute),
+      uri: `${process.env.END_POINT}`, // Server URL (must be absolute),
```

Comment:
I found the following description in the [official document](https://www.apollographql.com/docs/link/links/http/#context), so I named it this way.

> uri: a string of the endpoint you want to fetch from